app/models/atlas.rb

```diff
@@ -0,0 +1,32 @@
+class Atlas < ActiveRecord::Base
+  belongs_to :taxon
+  has_many :exploded_atlas_places
+  has_many :atlas_alterations
```

Comment:
Both of these `has_many` relations need `dependent: :delete_all` and `inverse_of:`